packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -115,6 +117,102 @@
   // stylelint-enable plugin/selector-bem-pattern
 }
 
+.mdc-text-field--outlined {
+  height: 56px;
+  border: none;
+
+  .mdc-text-field__outline {
```

Comment:
Isn't this redundant, and can't this be moved into the outline subcomponent's scss? i.e. you're not going to have this element within anything besides a `.mdc-text-field--outlined` so this doesn't need to be nested within that selector.

Reply:
Whoops, accidentally added ripple stuff here. Will remove and put into the next PR to keep this one clean, and yep I agree it should've been in the subcomponent's scss instead of here anyways.